<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Group A Steered Research Project</title>

    <!-- Bootstrap Core CSS -->
    <link href="CSS/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="CSS/4-col-portfolio.css" rel="stylesheet">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <body background="Images/geometry.png">
    <!-- Navigation -->
    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="navbar-brand" href="index.html">Project Webserver</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="background.html">Analysis Summary</a>
                    </li>
                    <li class="dropdown"><a class="dropdown-toggle" data-toggle="dropdown" href="#">Data<span class="caret"></span></a>
                         <ul class="dropdown-menu">
                              <li><a href="gene_search.php">Expression Levels</a></li>
                              <li><a href="diff_exp.php">Differential Expression</a></li>
                              <li><a href="graph_data.html">Volcano Plot with R Shiny</a></li>
                              <li><a href="search_ucsc.php">Visualise Mapped Data</a></li>
                              <li><a href="fastqc.html">View FastQC Files</a></li>
                         </ul>
                    </li>
                    <li>
                        <a href="team.html">Team</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Page Content -->
    <div class="container">

        <!-- Page Heading -->
        <div class="row">
            <div class="col-lg-12">
                <h1 class="page-header">Summary of Our Analysis
                </h1>

		<article>
		<font size="3">
		<p align="justify">
		    A comparative analysis was performed by Wang <em>et al. </em>to observe the
		    difference between RNA-sequencing (RNA-seq) and microarray in the detection
		    of differentially expressed genes (DEGs). In their study, they used 105 rat
		    livers that were exposed to 27 drugs with different modes of action to
		    identify differentially expressed levels of genes. In our study, we tried
		    to replicate the same analysis using one of the drugs they they analysed in
		    their study to see whether we could achieve the similar results.
		</p>
		<p align="justify">
		    Of the 27 drugs, we decided to choose the drug gemfibrozil (GEM) because
		    its mode of action, PPARA, demonstrated a modest gene expression
		    predictability. Furthermore, when quantitative polymerase chain reaction
		    (qPCR) was used to confirm the results of 18-cancer related genes, it was
		    found that those rats subjected to GEM, compared to the other test drug
		    sets, had not expressed a below-median cancer gene. For our controls, we
		    used the two matched controls that was run over the same 7-day duration as
		    GEM.
		</p>
		<p align="justify">
		    Prior to beginning the analysis, we performed a quality check on the
		    paired-end sequences we obtained from National Centre for Biotechnology
		    Information’s sequence read archive<sup>1</sup> on Wang <em>et al.’</em>s
		    study. We used the tool FastQC<sup>2</sup> to assess which reads needed to
		    be trimmed or excluded from the study.
		</p>
		<p align="justify">
		    We decided to use two pipelines to carry our analysis: Pipeline 5 (P5):
		    University of California, Santa Cruz (UCSC)<sup>3</sup>-TopHat<sup>4</sup>
		-Cufflinks<sup>5</sup> and Pipeline 6 (P6): Ensembl<sup>6</sup>-Bowtie    <sup>7</sup>-RNA-seq by Expectation Maximisation (RSEM)<sup>8</sup>. We
		    chose these two pipelines to determine whether using two different gene
		    models and two different approaches to mapping and quantifying transcripts
		    could produce similar results. Both pipelines were carried out using the
		    same rat genome “Rn4” that Wang <em>et al.</em> had used in their study and
		    on the latest rat genome “Rn6”.
		</p>
		<p align="justify">
		    P5’s TopHat and P6’s Bowtie were tools used to align mapped reads to the
		    genome before proceeding to the quantification of transcripts. Using the
		    webserver, users could view where these reads were mapped to genome using
		    the UCSC browser<sup>3</sup>.
		</p>
		<p align="justify">
		    One of the obvious differences observed between the two pipelines was how
		    the expression levels of genes were quantified. Cufflinks output values
		    were in the Fragments Per Kilobase of exon per Million fragments mapped
		    (FPKM) units, while RSEM gave raw counts. The difference between FPKM and
		    counts was that FPKM outputs were already normalised according to the
		    transcript length, while raw counts had to still be normalised using
		    statistical methods.
		</p>
		<p align="justify">
		    The resulting genes identified from these two pipelines were put through
		    the statistical tools, LIMMA<sup>9</sup> and edgeR<sup>10</sup>, for P5 and
		    P6, respectively. The log fold change (FC) and p-value for each gene was
		    calculated. The gene was determined to be significantly expressed when the
		    absolute logFC was &gt;1.5 and the p.value was &lt;0.05. These results
		could then be visualised using a volcano plot using the R package    <sup>12</sup>. A added feature to this volcano was added so that a user
		    could select the number of top DEGs they wanted to view.
		</p>
		<p align="justify">
		    <b>Bibliography</b>
		</p>
		<p align="justify">
		    1. Leinonen, R., Sugawara, H. and Shumway, M. (2011) 'The Sequence Read
		    Archive', <em>Nucleic Acids Research, </em>39(Database), pp. D21.
		</p>
		<p align="justify">
		    2. Andrews, S: FastQC, a Quality Control Tool for high-throughput Sequence
		    Data”,
		    <a
			href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
			target="_blank"
		    >
			http://www.bioinformatics.babraham.ac.uk/projects/fastqc/
		    </a>
		    (2010)
		</p>
		<p align="justify">
		    3. Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, Haussler
		D: The human genome browser at UCSC. <em>Genome research </em>2002, 12(6)    <strong>;</strong>996-1006.
		</p>
		<p align="justify">
		    4. Trapnell C, Pachter L, Salzberg SL: TopHat: discovering splice junctions
		    with RNA-Seq. <em>Bioinformatics </em>2009, 25(9)<strong>;</strong>
		    1105-1111.
		</p>
		<p align="justify">
		    5. Trapnell C, Williams BA, Pertea G, Mortazavi A, Kwan G, van Baren MJ,
		    Salzberg SL, Wold BJ, Pachter L: Transcript assembly and quantification by
		    RNA-Seq reveals unannotated transcripts and isoform switching during cell
		differentiation. <em>Nature Biotechnology </em>2010, 28(5)    <strong>;</strong>511-515.
		</p>
		<p align="justify">
		    6. Yates A, Akanni W, Amode MR, Barrell D, Billis K, Carvalho-Silva D,
		    Cummins C, Clapham P, Fitzgerald S, Gil L, Girón CG, Gordon L, Hourlier T,
		    Hunt SE, Janacek SH, Johnson N, Juettemann T, Keenan S, Lavidas I, Martin
		    FJ, Maurel T, McLaren W, Murphy DN, Nag R, Nuhn M, Parker A, Patricio M,
		    Pignatelli M, Rahtz M, Riat HS, Sheppard D, Taylor K, Thormann A, Vullo A,
		    Wilder SP, Zadissa A, Birney E, Harrow J, Muffato M, Perry E, Ruffier M,
		    Spudich G, Trevanion SJ, Cunningham F, Aken BL, Zerbino DR, Flicek P:
		Ensembl 2016. <em>Nucleic acids research </em>2016, 44(D1)    <strong>;</strong>D716.
		</p>
		<p align="justify">
		    7. Langmead B, Trapnell C, Pop M, Salzberg SL: Ultrafast and
		memory-efficient alignment of short DNA sequences to the human genome.    <em>Genome biology </em>2009, 10(3)<strong>;</strong>R25.
		</p>
		<p align="justify">
		    8. Li B, Dewey CN: RSEM: accurate transcript quantification from RNA-Seq
		    data with or without a reference genome. <em>BMC bioinformatics </em>2011,
		    12(1)<strong>;</strong>323.
		</p>
		<p align="justify">
		    9. Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK: limma powers
		differential expression analyses for RNA-sequencing and microarray studies.    <em>Nucleic acids research </em>2015, 43(7)<strong>;</strong>e47.
		</p>
		<p align="justify">
		    10. Robinson MD, McCarthy DJ, Smyth GK: edgeR: a Bioconductor package for
		differential expression analysis of digital gene expression data.    <em>Bioinformatics </em>2010, 26(1)<strong>;</strong>139-140.
		</p>
		<p align="justify">
		    11. R Development Core Team (2011), R: A Language and Environment for
		    Statistical Computing. Vienna, Austria : the R Foundation for Statistical
		Computing. ISBN: 3-900051-07-0. Available online at    <a href="http://www.r-project.org/">http://www.R-project.org/</a>.
		</p>

		</font>
		</article>

            </div>
        </div>
        <!-- /.row -->

        <hr>

        <!-- Footer -->
        <footer>
            <div class="row">
                <div class="col-lg-12">
                    <p>Group A Steered Research Project - MSc Bioinformatics @ University of Leicester - 2017</p>
                </div>
            </div>
            <!-- /.row -->
        </footer>

    </div>
    <!-- /.container -->

    <!-- jQuery -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

</body>

</html>
